tests/lammps/test_control.py

```diff
@@ -45,6 +46,14 @@ def test_mean(self):
         with self.assertRaises(NotImplementedError):
             lc.measure_mean_value('something')
 
+    def test_pressure_to_lammps(self):
+        lc = LammpsControl()
+        # Correct normalization without rotation. Note that we convert from GPa to bar for LAMMPS.
+        no_rot = np.identity(3)
+        self.assertEqual(lc.pressure_to_lammps(1.0, no_rot), [10000.0, 10000.0, 10000.0, None, None, None])
+        self.assertEqual(lc.pressure_to_lammps([1.0, 2.0, 3.0], no_rot), [10000.0, 20000.0, 30000.0, None, None, None])
+        self.assertEqual(lc.pressure_to_lammps([1.0, 2.0, 3.0, None, None, None], no_rot), [10000.0, 20000.0, 30000.0, None, None, None])
+        self.assertEqual(lc.pressure_to_lammps([None, None, None, None, None, 2.0], no_rot), [None, None, None, None, None, 20000.0])
 
```

Comment:
I apologize in advance if I'm missing something -- I looked over the diff carefully but I'm getting tired and might just be being dumb.

It looks to me like we allow using rotations in conjunction with fully defined pressure lists, but here there is no test for that, i.e. `pressure_to_lammps([a,b,c,d,e,f], some_rot)`. Don't we also need tests for this?

Reply:
There were no tests at all before, we should add a complete set before merging. But my brain capacity is diminishing today.